### ISSUE TITLE
Qt 6 support

### ISSUE THREAD

**Issue by jpgill86** [NeuralEnsemble/ephyviewer#146]
This is a thread for discussion of adding Qt 6 support to ephyviewer.

Regarding documentation for [QtPy](https://pypi.org/project/QtPy/), I found this docstring: https://github.com/spyder-ide/qtpy/blob/673fb93f5c126c6af1c2cb455da95c7e8b82cc62/qtpy/__init__.py#L9-L63

It seems `from qtpy import QtGui, QtWidgets, QtCore` will search for and import installed Qt interfaces in this order: PyQt5,
PySide2, PyQt4, PySide. As you pointed out, @samuelgarcia, there is an unmerged pull request to add PySide6 support (https://github.com/spyder-ide/qtpy/pull/225).

I'm still not sure if `from qtpy import QtGui, QtWidgets, QtCore` automatically handles API changes between Qt 4 and 5. I tried to test this, but I can't even manage to create a working environment with PyQt4 or PySide for testing Qt 4. For some reason, both of these fail with unexplained package conflicts: `conda create -n pyqt4 -c conda-forge pyqt=4` and `conda create -n pyside -c conda-forge pyside`. I'm having no better luck with pip. ¯\\\_\(ツ)\_/¯

**Comment by CAM-Gerlach** [comment]
Hey, it seems conda (including both Anaconda/AD/defaults and Miniconda/Conda-Forge) has apparently dropped support for the legacy Qt4 packages, as it seems I found out in spyder-ide/qtpy#208 . It looks like we'll have to drop CI testing of Qt4 in QtPy 1.10, which hopefully we can get out ASAP with the accumulated changes since the last release, though still supporting Qt4 on a best effort basis. The tentative plan I've floated (if others agree) is to then release 2.0 soon after, officially dropping Qt4 and adding PyQt6 and PySide6 support to QtPy. Or, we could include spyder-ide/qtpy#225 in 1.10 so it supports all three versions, if there is still significant value in supporting all three in one version before we drop Qt4, and we can find people on various platforms who can run the test suite locally to confbut I really don't know, that's above my purview.
